modules/assim.batch/vignettes/MultiSitePDAVignette.Rmd

```diff
@@ -0,0 +1,646 @@
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>2001</start.year>
+   <end.year>2014</end.year>
+  </settings.3>
+  <settings.4>
+   <quantiles>
+    <sigma>-1</sigma>
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>2005</start.year>
+   <end.year>2015</end.year>
+  </settings.4>
+  <settings.5>
+   <quantiles>
+    <sigma>-1</sigma>
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>2007</start.year>
+   <end.year>2014</end.year>
+  </settings.5>
+  <settings.6>
+   <quantiles>
+    <sigma>-1</sigma>
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>1999</start.year>
+   <end.year>2006</end.year>
+  </settings.6>
+  <settings.7>
+   <quantiles>
+    <sigma>-1</sigma>
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>2013</start.year>
+   <end.year>2015</end.year>
+  </settings.7>
+  <settings.8>
+   <quantiles>
+    <sigma>-1</sigma>
+    <sigma>0</sigma>
+    <sigma>1</sigma>
+   </quantiles>
+   <variable>NEE</variable>
+   <ensemble.id>1000017325</ensemble.id>
+   <start.year>1997</start.year>
+   <end.year>2010</end.year>
+  </settings.8>
+ </sensitivity.analysis>
+```
+
+## Settings for HB-PDA
+
+Similar to PDA settings, we will contain HB-PDA settings under the `<assim.batch>` section. Likewise standard PDA, if you have chosen the parameters you want to target in the calibration, open the `pecan.CONFIGS.xml` file, insert the tag below, and save as `pecan.HBPDA.xml`:
+
+```
+<assim.batch>
+  <method>emulator.ms</method>
+  <mode>local</mode>
+  <n.knot>60</n.knot>
+  <iter>10000</iter>
+  <chain>3</chain>
+  <param.names>
+   <soil>
+    <param>som_respiration_rate</param>
+    <param>soil_respiration_Q10</param>
+    <param>soilWHC</param>
+   </soil>
+   <temperate.deciduous>
+    <param>half_saturation_PAR</param>
+    <param>dVPDSlope</param>
+    <param>psnTOpt</param>
+   </temperate.deciduous>
+  </param.names>
+</assim.batch>
+```
+
+Different from the PDA settings, this time the main function will be the `emulator.ms`. This function can be run in three modes, which will be selected under the `<mode>` tag above: local, global and hierarchical. If left empty, the function will go through all three modes. But note that, for both global and hierarhical calibration, we need site-level runs. We will start with local mode, i.e. site-level calibration. 
+
+Now we will extend the settings above to include information about data to assimilate and likelihoods, similar to the standard-PDA setup, except for multiple sites. First, add `<assim.batch>` tag to the multisettings in the bottom of your `pecan.HBPDA.xml`:
+
+```
+ <multisettings>
```

Comment:
I don't recall this tag before, are you adding it?

Reply:
this is also automatically being added by pecan workflow at the end of the CONFIGS.xml.  I added the tag below so that multi.settings changes also apply to `assim.batch` tags in the xml.

So, at the end of CONFIGS.xml there was this chunk (created by the workflow):
```
 <multisettings>
  <multisettings>ensemble</multisettings>
  <multisettings>sensitivity.analysis</multisettings>
  <multisettings>run</multisettings>
 </multisettings>
```
If the module is listed there, `read.settings` reads the tag as `multi.settings` for that module. As assim.batch tags are added manually currently, so I also added `<multisettings>assim.batch</multisettings>` there manually.

Reply:
Hehe. Yeah, maybe not the best naming choice I've made. But yes, this is just a bit of metadata the MultiSettings code uses to know which blocks are meant to be replicated for each Settings. Looks kludgey to me (even aside from the naming). But I vaguely remember being convinced there needed to be something in there to make it explicit which settings were to be expanded.